### PR TITLE
Added response with string

### PR DESCRIPTION
**Pull request by tomasJancar**

### CODE REVIEW

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 1]
```diff
@@ -265,7 +305,27 @@ protected function expectRequestFail(
...
+    }
+
+
+    /**
+     * @param mixed[] $requestOptions
+     */
+    protected function expectRequestWithStringResponseFail(
+        string $method,
+        string $url,
+        int $errorCode = 400,
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:324]: StatusCode::

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 2]
```diff
@@ -230,6 +249,27 @@ public function expectGoldenPlatformRequestFail(
...
 
 
+    /**
+     * @param mixed[] $requestOptions
+     */
+    protected function expectAuthorizedRequestWithStringResponseFail(
+        string $method,
+        string $url,
+        string $bearerToken,
+        int $errorCode = 400,
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:259]: StatusCode::

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 3]
```diff
@@ -120,7 +120,20 @@ protected function expectRequest(
...
+    }
+
+
+    /**
+     * @param mixed[]|null $requestOptions
+     */
+    protected function expectRequestWithStringResponse(
+        string $method,
+        string $url,
+        ?string $responseBody = null,
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:133]: myslim ze by to melo byt spise `string $responseBody = ''`

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 4]
```diff
@@ -230,6 +249,27 @@ public function expectGoldenPlatformRequestFail(
...
 
+    /**
+     * @param mixed[] $requestOptions
+     */
+    protected function expectAuthorizedRequestWithStringResponseFail(
+        string $method,
+        string $url,
+        string $bearerToken,
+        int $errorCode = 400,
+        ?string $responseBody = null,
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:260]: same here

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 5]
```diff
@@ -265,7 +305,27 @@ protected function expectRequestFail(
...
+
+
+    /**
+     * @param mixed[] $requestOptions
+     */
+    protected function expectRequestWithStringResponseFail(
+        string $method,
+        string $url,
+        int $errorCode = 400,
+        ?string $responseBody = null,
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:325]: and here

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php [hunk 6]
```diff
@@ -265,7 +305,27 @@ protected function expectRequestFail(
...
+        ?array $requestOptions = null
```
Comment (dominikvoda) [inline src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:326]:
```suggestion
        array $requestOptions = []
```

**Review by dominikvoda — APPROVED** [review]

File: src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php (hunk 1 above)
Comment (tomasJancar) [reply src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php:324]: This is not an SDK.

**Review by dominikkaluza — APPROVED** [review]

**Pull request merged**